src/couch_replicator.erl

```diff
@@ -129,3 +135,53 @@ cancel_replication(RepId, #user_ctx{name = Name, roles = Roles}) ->
             {error, not_found}
         end
      end.
+
+
+-spec stream_terminal_docs_info(binary(), user_doc_cb(), any()) -> any().
+stream_terminal_docs_info(Db, Cb, UserAcc) ->
+    DDoc = <<"_replicator">>,
+    View = <<"terminal_states">>,
+    QueryCb = fun handle_replicator_doc_query/2,
+    Args = #mrargs{view_type = map, reduce = false},
+    Acc = {Db, Cb, UserAcc},
+    try fabric:query_view(Db, DDoc, View, QueryCb, Acc, Args) of
+    {ok, {Db, Cb, UserAcc1}} ->
+        UserAcc1;
+    {not_found, missing} ->
+        UserAcc
```

Comment:
Same idea as `error:database_does_not_exit`, since responses are streamed back after we already replied `200 OK` and are just sending back the chunks, it seems simpler to not treat a missing replicator db as fatal, interrupt the streaming of json, and force showroom to handle it. It just skips over, and never calls the callback function, and returns UserAcc (which in our case chttpd's response) so caller can keep calling stream* for other kinds of docs (active as well).

https://github.com/cloudant/showroom/pull/73/commits/998f80f6a995e39a4aae1330827f2d20645909e4#diff-3a94a6e59c9ceb6cdc9825e80d7f0b5fR125